manifests/cf-manifest/build_manifest.sh

```diff
@@ -1,20 +1,18 @@
 #!/bin/sh
 
 set -e
-cd $(dirname $0)
 
-terraform_outputs=${TERRAFORM_OUTPUTS:-"outputs/terraform-outputs.yml"}
```

Comment:
I think the commit 58b8032e30c2d2ff8a497410ec536c695 should be split in two, as these changes are not related to the director uuid change